.github/workflows/nightly-pipeline.yaml

```diff
@@ -294,8 +295,42 @@ jobs:
           github_user: GPUtester
           workflow_file_name: test.yaml
           ref: ${{ fromJSON(needs.get-run-info.outputs.obj).branch }}
-          wait_interval: 30
+          wait_interval: 120
           client_payload: ${{ toJSON(fromJSON(needs.get-run-info.outputs.obj).payloads.cuxfilter) }}
           propagate_failure: false
           trigger_workflow: true
           wait_workflow: true
+  dask-cuda-build:
```

Comment:
Should we try to sort the jobs alphabetically or topologically (dependency order)?

Reply:
I thought about it, but I don't think it matters much.

Maybe as a future enhancement, but not a huge deal for now.